app/presenters/candidates/school_presenter.rb

```diff
@@ -56,7 +56,7 @@ def dress_code?
     def formatted_dress_code
       return unless dress_code?
 
-      simple_format [dress_code, dress_code_other_details].join("\n\n")
+      simple_format strip_tags [dress_code, dress_code_other_details].join("\n\n")
```

Comment:
should this be `safe_format`?

Reply:
Yeh, did debate that - safe_format is in ApplicationHelper and I didn't want to pull the whole of that in. Could split it out into its own module and include that?